TOMP-API.yaml

```diff
@@ -996,29 +1001,21 @@ paths:
         '401':
           $ref: '#/components/responses/401Unauthorized'
 
-  /support/{id}/request/{support-type}:
+  /support/:
```

Comment:
What is the motivation of putting this in the develop-minor branch, when it seems like a breaking change? Is that how you want to split 0.6 vs 1.0?

Reply:
Nope. I try to do it in 2 steps.. 0.6 in next meeting, 1.0 in the meeting of june.